src/circle_marker/index.scss

```diff
@@ -0,0 +1,31 @@
+.c-circle_marker-container {
+  @include block-center;
+
+  width: 160px;
+  height: 160px;
+  padding: 15px;
+
+  text-align: center;
+  color: $color-gray-09;
+
+  border-radius: 50%;
+  border: 3px dotted $color-gray-02;
+  background: alpha($color-green-02);
```

Comment:
https://sass-lang.com/documentation/modules/color
Returns the alpha channel of $color as a number between 0 and 1.

use  `rgba`